index.js

```diff
@@ -0,0 +1,197 @@
+
+  reOrder();
+  loadItems();
+
+  document.getElementById('input').value = '';
+}
+
+function updateItem() {
+  const items = JSON.parse(localStorage.getItem('items'));
+
+  const updatedValue = document.getElementById('updateId').value;
+  const itemIndex = document
+    .getElementById('updateId')
+    .getAttribute('elementid');
+
+  items[itemIndex].description = updatedValue;
+
+  localStorage.setItem('items', JSON.stringify(items));
+
+  loadItems();
+
+  document.getElementById('formCreate').style.display = 'block';
+  document.getElementById('formUpdate').style.display = 'none';
+}
+
+function checkStatus() {
+  const items = JSON.parse(localStorage.getItem('items'));
+  const checkBoxes = document.querySelectorAll('.check');
+
+  checkBoxes.forEach((box, index) => {
+    const paragraph = box.nextElementSibling;
+
+    box.checked = items[index].completed;
+    if (items[index].completed) {
+      paragraph.style.textDecoration = 'line-through';
+      paragraph.style.color = '#999';
+    } else {
+      paragraph.style.textDecoration = 'none';
+      paragraph.style.color = 'black';
+    }
+
+    box.addEventListener('click', () => {
+      items[index].completed = box.checked;
+      if (items[index].completed) {
+        paragraph.style.textDecoration = 'line-through';
+        paragraph.style.color = '#999';
+      } else {
+        paragraph.style.textDecoration = 'none';
+        paragraph.style.color = 'black';
+      }
+      localStorage.setItem('items', JSON.stringify(items));
+    });
+  });
+}
+
+function checkNull(item) {
+  return item.description !== null;
+}
+
+function deleteItem() {
+  const items = JSON.parse(localStorage.getItem('items'));
+
+  const deleteBtnz = document.querySelectorAll('.trash');
+  deleteBtnz.forEach((button, index) => {
+    button.addEventListener('click', (e) => {
+      delete items[index];
+      const clean = items.filter(checkNull);
+      localStorage.setItem('items', JSON.stringify(clean));
+      reOrder();
+      e.target.parentElement.style.display = 'none';
+    });
+  });
+}
+
+function editItem() {
+  const editItems = document.querySelectorAll('.fa-edit');
+
+  editItems.forEach((item, index) => {
+    item.addEventListener('click', () => {
+      document.getElementById('formCreate').style.display = 'none';
+      document.getElementById('formUpdate').style.display = 'block';
+      document.getElementById('updateId').setAttribute('elementid', index);
+
+      const previousSibling = item.previousElementSibling;
+
+      document.getElementById('updateId').value = previousSibling.textContent;
+    });
+  });
+}
+
+
+function loadItems() {
+  const listContainer = document.querySelector('.list');
+
+  listContainer.innerHTML = '';
+
+  const tasks = [];
+
+  if (localStorage.getItem('items').length > 0) {
+    const items = JSON.parse(localStorage.getItem('items'));
+    items.forEach((item) => {
+      tasks.push(item);
+    });
+  }
```

Comment:
- [ ] Please consider using logical OR in order to follow the KISS principle to keep your code simple and clean as much as you can. For example, you can achieve the above functionality with the following line of code: 
`const tasks = JSON.parse(localStorage.getItem('items')) || [];`

Reply:
Thanks for that observation 👍